docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.8"
+
+services:
+  frontend:
+    build: ./front-end
+    ports:
+      - 4200:80
+    depends_on:
+      - backend
+
+  backend:
+    build: ./back-end
+    ports:
+      - 3000:3000
+    environment:
+      DB_URL: mongodb://127.0.0.1:27017/todo/tasks
```

Comment:
Well the values that we wanted to mask with the .env file are present here as well. If was thinking about setting these values from environment variables, but I'm not sure that we need this line even. Do we really need to set the `DB_URL` in the compose file?

Reply:
I can see a probability that we do, and I also got a new idea. You should first merge the variables in the .env file into a DB_URL variable, we don't need 3 variables for the same cause anyways. Then (if you need) you could read the same `.env` file in the compose file as the environment. Just like here: https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute